blimflix/client/app/directives/topbar/topbar.directive.html

```diff
@@ -0,0 +1,27 @@
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="col-lg-3 header-size">
+    <div class="navbar-header navbar-size">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="blimflix-logo-link" href="#">
+        <img class="blimflix-logo" src="/images/blimflix-logo.png" alt="" />
+      </a>
+    </div>
+  </div>
+  <div class="col-lg-9">
+    <div id="navbar" class="navbar-collapse collapse ">
+      <ul class="nav navbar-nav list-size">
+        <li ng-repeat="section in sections" class="nav-link-right">
+          <a href="#">{{section.title}} <span class="badge badge-color">{{section.count}}</span></a>
```

Comment:
Please use [ng-href directive](https://docs.angularjs.org/api/ng/directive/ngHref) and use the **section's url** as destination:

`
<a ng-href="{{section.url}}">{{section.title}} <span class="badge badge-color">{{section.count}}</span></a>
`